core/src/main/kotlin/com/google/container/tools/core/properties/PropertiesFileFlagReader.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.container.tools.core.properties
+
+import java.util.Properties
+
+/**
+ * Reads property values from the specified properties file.
+ *
+ * @param propertyFilePath the path to the properties file; defaults to 'config.properties' located
+ * in the root of the module's resources
+ */
+class PropertiesFileFlagReader(
```

Comment:
I feel this name is a bit confusing - it mentions `flags`, but all javadocs just say "property". Property looks to be very familiar concept, what if we just use property and maybe also hint this is out plugin properties, not some general purpose class - i.e. `ContainerToolsProperties` or something similar?

Reply:
Completely agree. How about just `PropertiesFileReader`?

Reply:
Sounds good! In case we only use this class as a source of properties for our plugin and don't anticipate to use as as general property reader, we can probably further customize its name as a plugin propeties something?